cmake/build_configurations/mysql_release.cmake

```diff
@@ -270,27 +278,33 @@ IF(UNIX)
       IF(CMAKE_SYSTEM_PROCESSOR MATCHES "i386")
         SET(COMMON_C_FLAGS                   "-g -mt -fsimple=1 -ftrap=%none -nofstore -xbuiltin=%all -xlibmil -xlibmopt -xtarget=generic")
         SET(COMMON_CXX_FLAGS                 "-g0 -mt -fsimple=1 -ftrap=%none -nofstore -xbuiltin=%all -features=no%except -xlibmil -xlibmopt -xtarget=generic")
-        SET(CMAKE_C_FLAGS_DEBUG              "-xO1 ${COMMON_C_FLAGS}")
-        SET(CMAKE_CXX_FLAGS_DEBUG            "-xO1 ${COMMON_CXX_FLAGS}")
```

Comment:
Solaris on i386 is dead, right? Even Oracle switched from Sun Forte C or SunPro to GCC at least for a while during the early phase of the MySQL 8.0 development cycle, because the C++11 support was not adequate.